Sources/PaymentsCore/Networking/Models/ErrorResponse.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+struct ErrorResponse: Codable {
+    let name: String
+    let message: String
```

Comment:
I see our API has a `details` field which contains very useful info about the error. Should we add it here?

Reply:
and I think `message` here can be optional? I see if I put in an invalid access token we won't have `message` :smile:

Reply:
Good call - we can make this optional.